### PR TITLE
Fix environment and permissions

### PR DESCRIPTION
**Pull request by renan**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,9 +3,10 @@ FROM alpine:3.9
 LABEL maintainer="technology@werkspot.com"
  
 RUN apk add --no-cache varnish
+RUN chown nobody:nogroup -R /etc/varnish
```
Comment (jeroenvdheuvel) [inline Dockerfile:6]: Why do we need to change ownership of this directory? In most cases, a volume should be mounted in this directory. The mounted directory can have any owner you'd like.

File: Dockerfile [hunk 2]
```diff
@@ -3,9 +3,10 @@ FROM alpine:3.9
...
 
-ENV LISTEN_ADDRESS ":8080" \
-    WORKING_DIRECTORY="/tmp/varnish"
+ENV LISTEN_ADDRESS ":8080"
+ENV WORKING_DIRECTORY "/tmp/varnish"
```
Comment (ruudvanderweijde) [inline Dockerfile:9]: This one is fixed in the other PR, which is already merged

Comment (renan) [reply Dockerfile:9]: :+1:

File: Dockerfile (hunk 1 above)
Comment (renan) [reply Dockerfile:6]: In this microservice we built we have a Dockerfile extending `werkspot/varnish` which does a `COPY` instruction, hence needing the right permission when executing `varnishd` with `/etc/varnish/default.vcl`

Comment (ruudvanderweijde) [reply Dockerfile:6]: Just tested this version and it works like a char

**Pull request merged**